src/main/java/io/odh/test/Environment.java

```diff
@@ -69,7 +69,9 @@ public class Environment {
     private static final String OPERATOR_INSTALL_TYPE_ENV = "OPERATOR_INSTALL_TYPE";
     private static final String OLM_UPGRADE_STARTING_VERSION_ENV = "OLM_UPGRADE_STARTING_VERSION";
 
-    public static final String PRODUCT_DEFAULT = "odh";
+    public static final String PRODUCT_ODH = "odh";
+    public static final String PRODUCT_RHOAI = "rhoai";
+    public static final String PRODUCT_DEFAULT = PRODUCT_ODH;
```

Comment:
In scope of this change I dong get why there is still product default property

Reply:
I was thinking to keep it here so that we can set default product on one place and use it on multiple places eventually. But yeah, we use it just on one place right now... and I think that we probably won't need it on another place. So good point, I'll remove.

Reply:
done